app/test/app.test.js

```diff
@@ -64,11 +64,22 @@ test('when post id is string', async () => {
 test('when id exists', async () => {
     let newPost = new postModel.PostEntity(null, 'title', 'slug', 'content');
     await postModel.insertPost(newPost);
-    
```

Comment:
https://i.stack.imgur.com/hZrUY.png

"trim trailing whitespace" też sobie włącz

Reply:
ogólnie to te 3 opcje

```
  "files.insertFinalNewline": true,
  "files.trimFinalNewlines": true,
  "files.trimTrailingWhitespace": true,
```